src/sentry/static/sentry/app/plugins/components/issueActions.tsx

```diff
@@ -299,15 +372,17 @@ class IssueActions extends PluginComponentBase {
 
     if (impactedField) {
       //if every dependent field is set, then search
-      if (!impactedField.depends.some(dependentField => !formData[dependentField])) {
+      if (!impactedField.depends?.some(dependentField => !formData[dependentField])) {
         callback = () => this.loadOptionsForDependentField(impactedField);
       } else {
         //otherwise reset the options
         callback = () => this.resetOptionsOfDependentField(impactedField);
       }
     }
-
-    this.setState({[formDataKey]: formData}, callback);
+    const newState = {[formDataKey]: formData} as {
+      [x in typeof formDataKey]: Record<string, any>;
+    };
+    this.setState(newState, callback);
   }
 
   renderForm() {
```

Comment:
`renderForm` is returning any because of the funky form variable being assigned

Reply:
@scttcper fixed